rspace/src/main/scala/coop/rchain/rspace/history/HistoryRepositoryImpl.scala

```diff
@@ -72,6 +72,8 @@ final case class HistoryRepositoryImpl[F[_]: Concurrent: Parallel: Log: Span, C,
 
   private def calculateStorageActions(action: HotStoreTrieAction): Result = {
     import HistoryRepositoryInstances.{PREFIX_DATUM, PREFIX_JOINS, PREFIX_KONT}
+    implicit def fromSeqToKeySegment(prefixSeq: Seq[Byte]): KeySegment =
+      KeySegment(prefixSeq.toArray)
```

Comment:
Hm :thinking: Why not to make this constructing **explicit** like other constructors in KeySegment companion object? Or use this approach with **implicit** function in other similar places?